### PR TITLE
Update devserver dockerfile to match changed skdb-base

### PR DESCRIPTION
**Pull request by bennostein**
Use renamed base image, and drop no-op copy which was causing errors.

### CODE REVIEW

**Comment by bennostein** [comment]
I've pushed latest+quickstart versions of skdb-base and skdb-dev-server using these changes, and NPM packages to match.  Everything seems good on my end, but shout if that's broken you.

File: sql/server/dev/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,6 @@ FROM eclipse-temurin:20 as dev
 
 WORKDIR /skdb
 
-COPY --from=skdb /skdb/build ./build
```
Comment (gregsexton) [inline sql/server/dev/Dockerfile:17]: We need skdb/build for the skdb binary that the dev server uses. Without this we don't have the binary.

File: sql/server/dev/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM skiplabs/skdb as skdb
+FROM skiplabs/skdb-base as skdb
```
Comment (gregsexton) [inline sql/server/dev/Dockerfile:1]: This should be skiplabs/skdb which builds on skiplabs/skdb-base. The dockerfile for this is in sql/Dockerfile. I refactored to make building for prod possible. I think I might have missed updating a script that needs to build this for the dev server?

**Review by gregsexton — CHANGES_REQUESTED** [review]

**Comment by gregsexton** [comment]
I imagine the dev server you built and pushed doesn't work if you try to connect to it?

**Comment by bennostein** [comment]
Yes, you're right.  I think I jumped the gun here with these changes after running into some unrelated issues building a new devserver image yesterday.  Spent most of yesterday afternoon trying to figure that out; will keep investigating today; closing this one now.

**Pull request closed without merging**